lib/miq_expression.rb

```diff
@@ -129,13 +129,13 @@ def self._to_human(exp, options = {})
       clause = "#{operands.first} #{operator} #{operands.last}"
     when "between dates", "between times"
       col_name = exp[operator]["field"]
-      col_type = get_col_type(col_name)
+      col_type = parse_field_or_tag(col_name).try(:column_type)
```

Comment:
@lpichler is `&.column_type`, `.try!(:column_type)` or `.try(:column_type)` the correct mechanism here?

Are we expecting an object (other than `nil`) to be returned that does not support`.column_type`?

Reply:
> @lpichler is `&.column_type`, `.try!(:column_type)` or `.try(:column_type)` the correct mechanism here?
> 
> Are we expecting an object (other than `nil`) to be returned that does not support`.column_type`?

No just, nil or MiqExpression::Field or CountField.... I need to investigate, whether it is valid to return nil from parse_field_or_tag  at all  because `nil` means from that the column is not valid - so it could not be processed by `MiqExpression` evaluation.

Reply:
and I used `&.column_type`